### PR TITLE
[terraform-conversions] Add compute_instance to conversions library

### PR DESCRIPTION
**Pull request by nstogner**
Add non-generated compute_instance to CAI conversions. Code was pulled from terraform provider `third_party/` and modified slightly.

### CODE REVIEW

**Comment by nstogner** [comment]
Unfortunately we are unable to use `*schema.ResourceData` struct instead of the `TerraformResourceData` interface in these conversions b/c there is no way to populate its unexported fields. An alternative would be to use the interface when possible in the provider. For the files that don't use `*schema.ResourceData` I will go ahead to source them from the `third_party/terraform/` directory rather than copying.

File: third_party/validator/compute_instance_helpers.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	computeBeta "google.golang.org/api/compute/v0.beta"
+)
+
+func expandAliasIpRanges(ranges []interface{}) []*computeBeta.AliasIpRange {
```
[the remark this answers is not in the document]
Comment (nstogner) [reply third_party/validator/compute_instance_helpers.go]: I am now just using the file from `third_party/terraform`. Do you want to remove the validation check in this PR or in another?

**Comment by nstogner** [comment]
Removed copying of files from `third_party/terraform/` to `third_party/validator/` except for the manual copy/patch of `compute_instance.go` itself.

File: third_party/validator/compute_instance.go [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+		return nil, fmt.Errorf("Error creating network interfaces: %s", err)
+	}
+
+	accels, err := expandInstanceGuestAccelerators(d, config)
+	if err != nil {
+		return nil, fmt.Errorf("Error creating guest accelerators: %s", err)
+	}
+
+	// Create the instance information
+	return &computeBeta.Instance{
```
Comment (chrisst) [inline third_party/validator/compute_instance.go]: We use the computeBeta library for legacy reasons. If you find that this becomes a problem you can probably move to the GA version.

**Comment by nstogner** [comment]
Signed CLA

**Pull request merged**